packages/ckeditor5-link/src/linkui.ts

```diff
@@ -211,7 +211,7 @@ export default class LinkUI extends Plugin {
 	private _createViews() {
 		this.toolbarView = this._createToolbarView();
 		this.formView = this._createFormView();
-		this.advancedView = this._createAdvancedView();
+		this.propertiesView = this._createPropertiesView();
```

Comment:
This view is created always, even when there are no decorators defined.

Reply:
Fixed https://github.com/ckeditor/ckeditor5/pull/17534/commits/d6e071dabce4307821cf8f9081b33558c48d7f25